src/lib/Containers/BidFlow.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import { NavigatorIOS, ViewProperties } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+
+import { SelectMaxBid } from "../Components/Bidding/Screens/SelectMaxBid"
+
+import { BidFlow_sale_artwork } from "__generated__/BidFlow_sale_artwork.graphql"
+
+interface BidFlowProps extends ViewProperties {
+  sale_artwork: BidFlow_sale_artwork
+}
+
+class BidFlow extends React.Component<BidFlowProps> {
+  render() {
+    return (
+      <NavigatorIOS
+        navigationBarHidden={true}
+        initialRoute={{
+          component: SelectMaxBid,
+          title: "", // title is required, though we don't use it because our navigation bar is hidden.
+          passProps: {
+            sale_artwork: this.props.sale_artwork,
+          },
+        }}
+        style={{ flex: 1 }}
+      />
+    )
+  }
+}
+
+export default createFragmentContainer(
+  BidFlow,
+  graphql`
+    fragment BidFlow_sale_artwork on SaleArtwork @relay(mask: false) {
+      bid_increments
+      ...SelectMaxBid_sale_artwork
```

Comment:
Okay so the two lines above are a problem. `...SelectMaxBid_sale_artwork` should spread on that fragment, which includes `bid_increments`. But if we remove `bid_increments` from the line above, they aren't fetched. It is as though the `SelectMaxBid_sale_artwork` fragment isn't being spread correctly.

I'll keep looking into this, but I haven't found any promising leads. @sarahscott @orta any ideas here?